### PR TITLE
Spatial, Transform and Transform2D documentation tweaks

### PR DESCRIPTION
**Pull request by SirPigeonz**

### CODE REVIEW

File: doc/classes/Spatial.xml [hunk 1]
```diff
@@ -110,14 +110,14 @@
...
-				Returns whether this node is set as Toplevel, ignoring its parent node transformations.
+				Returns whether this node is set as Toplevel (whether ignores its parent nodes transformations).
 			</description>
 		</method>
 		<method name="is_transform_notification_enabled" qualifiers="const">
 			<return type="bool">
 			</return>
 			<description>
-				Returns whether node sends notification that its transformation changed. Spatial will not propagate this by default.
+				Returns whether node sends notification that its transformation changed, global or local. Spatial will not propagate this by default.
```
Comment (akien-mga) [inline doc/classes/Spatial.xml]: "whether *the* node".

A clearer sentence would be "Returns whether the node notifies about its global and local transformation changes. Spatial will not propagate this notification by default."

File: doc/classes/Spatial.xml [hunk 2]
```diff
@@ -110,14 +110,14 @@
 			<return type="bool">
 			</return>
 			<description>
-				Returns whether this node is set as Toplevel, ignoring its parent node transformations.
+				Returns whether this node is set as Toplevel (whether ignores its parent nodes transformations).
```
Comment (akien-mga) [inline doc/classes/Spatial.xml]: I don't understand the parenthesis. Do you mean something like:
"Returns whether this node is set as "top level", i.e. it ignores its parent nodes' transformations."

Comment (n-pigeon) [reply doc/classes/Spatial.xml]: Yup. "i.e." is better idea tho. Will update.

File: doc/classes/Spatial.xml (hunk 1 above)
Comment (n-pigeon) [reply doc/classes/Spatial.xml]: Sounds good.

**Comment by n-pigeon** [comment]
@akien-mga pushed changes.

**Pull request merged**